### PR TITLE
Move telemetry out of pilot

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

File: pkg/apis/istio/v1alpha1/values_types.proto
Comment (richardwxn) [inline pkg/apis/istio/v1alpha1/values_types.proto]: This should be TelemetryV2Config?

Comment (howardjohn) [reply pkg/apis/istio/v1alpha1/values_types.proto]: Yes, and blocked by https://github.com/istio/installer/pull/551. I can update this once that merges

**Review by ostromart — APPROVED** [review]

**Pull request merged**
